pyop2/codegen/rep2loopy.py

```diff
@@ -535,6 +546,14 @@ def renamer(expr):
         assumptions = assumptions & pwaffd[parameters.layer_start].le_set(pwaffd[parameters.layer_end])
     assumptions = reduce(operator.and_, assumptions.get_basic_sets())
 
+    # Even though we think we are setting loop priorities correctly here, loopy cannot
+    # use its new scheduler and raises a warning stating that there are
+    # "loop priority dependencies between sibling loop nests".
+    # See https://github.com/inducer/loopy/issues/890.
+    # Therefore until loopy fixes this we disable this warning and use the old (slower)
+    # scheduling algorithm.
```

Comment:
Please open a Firedrake issue linking to these lines and the loopy issue to we can keep track and hopefully revery once loopy gets fixed.

Reply:
Won't be long! https://github.com/inducer/loopy/pull/895
I might just wait for that to be merged.

Reply:
Fixed upstream. Will remove this.